src/ChartTemplates/StackChart/StackChartTemplate.js

```diff
@@ -52,23 +48,25 @@ export const StackChart = ({ ...props }) => {
                 />)}
             </ChartStack>
         </Chart>
-        <span className='stackChartLegend'>
+        <span className='stackChartLegend' aria-label="Chart legend">
+            <table tabIndex="0" className="visually-hidden" aria-label={ props.ariaTitle + ` data` }>
+                { props.data.map((d, index) => {
+                    return [
+                        <tr key={ index }>
+                            <td>{ d.y }</td>
+                            <td>{ d.name }</td>
+                        </tr>
+                    ];
+                }) }
+            </table>
             <ChartLegend
                 data={ legendData }
                 responsive={ false }
                 height={ props.legendHeight }
                 width={ props.legendWidth }
                 fontSize={ chartLegendFontSize }
                 className='pf-m-redhat-font'
-                style={ {
```

Comment:
M not so sure about this change... it spawns two extra files and a buncha lines of code... does any behavior change?

Reply:
I spoke to Dan, and this change allows us to just pass in the StackChartLabel component and remove all the props for `style`, `labels` and `events`, because now the label is a button. This is the way we want to handle links inside of labels in charts from now on.

Do you think I should move the label file into this file to reduce the number of files?

Reply:
Unless the component gonna be used in another file, yeah, keep everything in the one...

Totally down with killin the events, n thats the way it oughta be done now that we have the <a> there.. but the styles... is there a reason why we wanta remove them? especially when it takes 3x the code and a few `!importants` to get it to work?

Reply:
Great point. I just tried playing around with the style though, and I don't know how to change the hover/focus/active styles through the `styles` prop, do you?

Ideally it should look like `pf-c-button pf-m-link pf-m-inline` like the screenshot below when it is in the H/A/F state. If we can do that through the `styles` prop then that would be good, but otherwise I think we should have the stylesheet as it would be easier to maintain.

<img width="847" alt="Screen Shot 2020-05-14 at 3 27 40 PM" src="https://user-images.githubusercontent.com/20118816/81977221-ae2f0800-95f7-11ea-8ddb-bacb19c35d64.png">

Reply:
OOO the difference that yer looking to fix being the color changes on hover? Yeah checkout the VictoryLegend documentation... specifically events... legend takes events n yah can do mutations n such https://formidable.com/open-source/victory/docs/victory-legend/#events

Reply:
Do you know if there is an event for `onHover` rather than `onClick` I've been playing around with the code but i can't seem to get the same implementation as just targeting the style sheet. ill keep working on it on Monday. I am a little hesitant though because for Accessibility reasons we will need the style change on `:hover` `:active` and `:focus` and I don't necessarily know that the events will cover all of these.

Reply:
looks like `onMouseOver` is in fact an event yah can mutate on https://formidable.com/open-source/victory/guides/events/#victorysharedevents


some other reading here
https://formidable.com/open-source/victory/docs/common-container-props#events

Reply:
updated!

<img width="650" alt="Screen Shot 2020-05-18 at 4 33 10 PM" src="https://user-images.githubusercontent.com/20118816/82257004-4a714b80-9925-11ea-8393-10e64c802581.png">

Reply:
Stellar!! looks great! last thing is to move that label piece back into the only file that uses it 👍  then we can merge thissssssss 🍕 🗡️

Reply:
updated :D